### PR TITLE
Update L2_Bridge_schema.graphql

### PR DESCRIPTION
**Pull request by PaulieB14**
Added descriptions.  *Note: copied from L1.  If I missed any differences please let me know.  But they looked the exact same.

### CODE REVIEW

File: L2_Bridge_schema.graphql [hunk 1]
```diff
@@ -1,410 +1,545 @@
...
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type MultipleWithdrawalsSettled @entity {
+type ChallengeResolved @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity shouldn't exist in L2 schema

File: L2_Bridge_schema.graphql [hunk 2]
```diff
@@ -1,410 +1,545 @@
...
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type TransferRootSet @entity {
+type TransferBondChallenged @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity shouldn't exist in L2 schema

File: L2_Bridge_schema.graphql [hunk 3]
```diff
@@ -1,410 +1,545 @@
...
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type TransferSent @entity {
+type TransferRootBonded @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity shouldn't exist in L2 schema

File: L2_Bridge_schema.graphql [hunk 4]
```diff
@@ -1,410 +1,545 @@
...
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type TransfersCommitted @entity {
+type TransferRootConfirmed @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity shouldn't exist in L2 schema

File: L2_Bridge_schema.graphql [hunk 5]
```diff
@@ -1,410 +1,545 @@
...
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type TransfersCommitted @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:144]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 6]
```diff
@@ -1,410 +1,545 @@
...
+  "Date unix timestamp"
+  date: Int!
+  "Bridge token asset entity"
   tokenEntity: Token!
 
   # legacy
-  token: String!
-}
-
-type BonderFee @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:405]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 7]
```diff
@@ -1,410 +1,545 @@
...
-type AmmFee @entity {
+type DailyVolume @entity {
+  "Entity ID"
   id: ID!
+  "Daily volume amount"
   amount: BigInt!
+  "Date unix timestamp"
+  date: Int!
+  "Bridge token asset entity"
   tokenEntity: Token!
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:675]: This entity property shouldn't exist

File: L2_Bridge_schema.graphql [hunk 8]
```diff
@@ -1,410 +1,545 @@
...
-}
 
-type DailyVolume @entity {
-  id: ID!
-  amount: BigInt!
-  token: String!
-  date: Int!
-}
+  "Bridge token asset entity"
+  tokenEntity: Token!
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity property shouldn't exist

File: L2_Bridge_schema.graphql [hunk 9]
```diff
@@ -1,410 +1,545 @@
...
-  transactionIndex: BigInt!
-  timestamp: BigInt!
-  blockNumber: BigInt!
-  contractAddress: String!
+  "From address"
   from: String!
-  token: String!
-}
-
-type RemoveLiquidityImbalance @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 10]
```diff
@@ -1,410 +1,545 @@
...
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type RemoveLiquidityOne @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:327]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 11]
```diff
@@ -1,410 +1,545 @@
...
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type RemoveLiquidity @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:307]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 12]
```diff
@@ -1,410 +1,545 @@
...
-  # legacy
-  transactionHash: String!
-  transactionIndex: BigInt!
-  timestamp: BigInt!
-  blockNumber: BigInt!
-  contractAddress: String!
-  token: String!
-}
-
-type AddLiquidity @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:285]: This entity shouldn't be removed

File: L2_Bridge_schema.graphql [hunk 13]
```diff
@@ -1,410 +1,545 @@
...
+  
+  "Contract address"
   contractAddress: String!
+  "From address"
   from: String!
+  "Bridge token asset symbol"
   token: String!
 }
 
-type TransferSent @entity {
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:118]: This entity shouldn't be removed

**Review by miguelmota — CHANGES_REQUESTED** [review]
Thanks for the entity descriptions. The L2 schema is not exactly the same as the L1 schema so copying the schema as is would cause undesired changes. Left some comments

**Comment by PaulieB14** [comment]
@miguelmota. Sorry I thought L2 was the same.  I went back and started from scratch and added descriptions.  There were some entities that were missing descriptions that I added.  Let me know if revisions are needed.

File: L2_Bridge_schema.graphql [hunk 14]
```diff
@@ -1,348 +1,588 @@
...
+  "Bonderfee"
   bonderFee: BigInt!
+  "Transaction index"
   index: BigInt!
+  "Minimum amount out"
   amountOutMin: BigInt!
+  "Deadline timestamp"
   deadline: BigInt!
-
+ƒ
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql]: there's a random artifact here

File: L2_Bridge_schema.graphql [hunk 15]
```diff
@@ -1,348 +1,588 @@
...
   transactionHash: String!
+  "Transaction index"
   transactionIndex: BigInt!
+  "Transaction timestamp"
   timestamp: BigInt!
+  "Transaction block number"
   blockNumber: BigInt!
+  "Contract address"
   contractAddress: String!
-  from: String!
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:262]: looks like this got overridden by mistake and should be added

File: L2_Bridge_schema.graphql [hunk 16]
```diff
@@ -1,348 +1,588 @@
...
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:323]: looks like this got overridden by mistake and should be added

File: L2_Bridge_schema.graphql [hunk 17]
```diff
@@ -1,348 +1,588 @@
...
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:345]: looks like this got overridden by mistake and should be added

File: L2_Bridge_schema.graphql [hunk 18]
```diff
@@ -354,57 +594,87 @@ type RemoveLiquidityImbalance @entity {
...
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:367]: looks like this got overridden by mistake and should be added

File: L2_Bridge_schema.graphql [hunk 19]
```diff
@@ -501,4 +771,8 @@ type Token @entity {
 
   "Token symbol"
   symbol: String!
-}
\ No newline at end of file
+}
+
```
Comment (miguelmota) [inline L2_Bridge_schema.graphql:785]: these trailing newlines should be removed

**Review by miguelmota — CHANGES_REQUESTED** [review]
thanks @PaulieB14! see additional minor comments that need to be addressed and then it can be merged

**Comment by PaulieB14** [comment]
@miguelmota Thanks for the revisions.  I think we are good to go now.

**Comment by miguelmota** [comment]
@PaulieB14 looks good! merging

**Pull request merged**